inginious/frontend/app.py

```diff
@@ -148,7 +150,15 @@ def get_app(config):
         task_directory = config["tasks_directory"]
         fs_provider = LocalFSProvider(task_directory)
 
-    course_factory, task_factory = create_factories(fs_provider, plugin_manager, WebAppCourse, WebAppTask)
+    default_problem_types = {
+        problem_type.get_type(): problem_type for problem_type in [DisplayableCodeProblem,
+                                                                   DisplayableCodeSingleLineProblem,
+                                                                   DisplayableCodeFileProblem,
+                                                                   DisplayableMultipleChoiceProblem,
+                                                                   DisplayableMatchProblem]
+    }
+
+    course_factory, task_factory = create_factories(fs_provider, default_problem_types, plugin_manager, WebAppCourse, WebAppTask)
```

Comment:
Just a side note: there should be a hook to add new types of problem here. It is problematic to use plugins as they are not initialized yet. Maybe we should use the same mechanism we use for the fs providers? (https://github.com/UCL-INGI/INGInious/blob/master/inginious/common/entrypoints.py). 

Ideally, we should also update the plugin system to discover plugins this way, but that's for another PR.